src/System.Security.Cryptography.Cng/src/Interop/BCrypt/Interop.AsymmetricEncryption.Types.cs

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+
+internal partial class Interop
+{
+    /// <summary>
+    /// BCrypt types related to asymmetric encryption algorithms
+    /// </summary>
+    internal partial class BCrypt
+    {
+        [StructLayout(LayoutKind.Sequential)]
+        internal struct BCRYPT_OAEP_PADDING_INFO
+        {
+            internal IntPtr pszAlgId;  // null-terminated Unicode string that identifies the hashing algorithm used to create the padding.
+            internal IntPtr pbLabel;   // address of a buffer that contains the data used to create the padding.
+            internal int cbLabel;      // number of bytes in the pbLabel buffer.
+        }
+
+        [StructLayout(LayoutKind.Sequential)]
+        internal struct BCRYPT_PKCS1_PADDING_INFO
+        {
+            internal IntPtr pszAlgId;  // null-terminated Unicode string that identifies the hashing algorithm used to create the padding.
+        }
+
+        [StructLayout(LayoutKind.Sequential)]
+        internal struct BCRYPT_PSS_PADDING_INFO
+        {
+            internal IntPtr pszAlgId;  // null-terminated Unicode string that identifies the hashing algorithm used to create the padding.
+            internal int cbSalt;       // The size, in bytes, of the random salt to use for the padding.
```

Comment:
If we're going to add comments for every field like this, should we do it as XML comments instead?  That helps to not only document it in code, but provides the information to the dev via IntelliSense while working on the code that uses the fields.